client/jira.go

```diff
@@ -72,6 +72,14 @@ type (
 		Issues       []domain.Issue `json:"issues"'`
 	}
 
+	TransitionIdRequest struct {
```

Comment:
Change to `transition`

Reply:
Actually most of them can be private lowercase

Reply:
done